### DISCUSSION TITLE
Enhance explore view to show Loki errors

### DISCUSSION THREAD

**Discussion by alanhe** [grafana/grafana#42317] [Observability]
**What would you like to be added**:
There's a `status` field in the Loki query response, which can be either "success" or "fail".
https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/loki/types.ts#L54

The value is ignored when parsing results. It will be better if the value fo `status` field can be shown in the Explore's `ResponseErrorContainer` when the value is not "success".

Give users more details of what's going on.

**Why is this needed**:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
